src/View/Helper/FormHelper.php

```diff
@@ -75,12 +75,12 @@ class FormHelper extends Helper
             'formGroup' => '{{label}}<div class="%s">{{input}}{{error}}{{help}}</div>',
             'checkboxFormGroup' => '<div class="%s"><div class="checkbox">{{label}}</div>{{error}}{{help}}</div>',
             'submitContainer' => '<div class="form-group"><div class="%s">{{content}}</div></div>',
```

Comment:
This should have `row` too I think.

Reply:
I haven't made any related to your issue with `FormHelper::submit()`. Why are you even using submit inputs instead of generating buttons using `FormHelper::button()`? :slightly_smiling_face:

Reply:
Old habits I guess, and the alignment with the inputs was visually more appealing :)